NitroxClient/GameLogic/InitialSync/PlayerInitialSyncProcessor.cs

```diff
@@ -21,18 +22,22 @@ public PlayerInitialSyncProcessor(ItemContainers itemContainers)
         public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualWaitItem waitScreenItem)
         {
             SetPlayerGameObjectId(packet.PlayerGameObjectId);
-            waitScreenItem.SetProgress(0.25f);
+            waitScreenItem.SetProgress(0.2f);
             yield return null;
 
             AddStartingItemsToPlayer(packet.FirstTimeConnecting);
-            waitScreenItem.SetProgress(0.5f);
+            waitScreenItem.SetProgress(0.4f);
             yield return null;
 
             SetPlayerStats(packet.PlayerStatsData);
-            waitScreenItem.SetProgress(0.75f);
+            waitScreenItem.SetProgress(0.6f);
             yield return null;
 
             SetPlayerGameMode(packet.GameMode);
+            waitScreenItem.SetProgress(.8f);
```

Comment:
Let's keep it consistent with the other progressions
```suggestion
            waitScreenItem.SetProgress(0.8f);
```

Reply:
Updated branch with the prefixed zero